kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/KubernetesListOperationsImpl.java

```diff
@@ -57,6 +58,9 @@ public ClientKubernetesListNonNamespaceOperation inNamespace(String namespace) {
   @Override
   public KubernetesList create(KubernetesList... items) {
     List<HasMetadata> createdItems = new ArrayList<>();
+    if (items.length == 0) {
+      items[0] = get();
```

Comment:
This isn't going to work. You'll need to do something like `items = new KubernetesList[]{get()};`